MapboxNavigation.xcodeproj/project.pbxproj

```diff
@@ -2317,12 +2328,16 @@
 			isa = PBXResourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				E27A20D02654EBED00AA935F /* li.tar in Resources */,
 				B4A78D19261FD47700FDF212 /* route-with-banner-instructions.json in Resources */,
 				B4A78D1A261FD47700FDF212 /* route-with-missing-road-classes.json in Resources */,
+				E27A20CF2654EBED00AA935F /* GGPark-to-BernalHeights.route in Resources */,
 				B4A78D1B261FD47700FDF212 /* route-with-instructions.json in Resources */,
 				B4A78D1C261FD47700FDF212 /* routeWithTunnels_9thStreetDC.json in Resources */,
 				B4A78D1D261FD47700FDF212 /* PipeFittersUnion-FourSeasonsBoston.json in Resources */,
+				E27A20D12654EBED00AA935F /* UnionSquare-to-GGPark.route in Resources */,
 				B4A78D1E261FD47700FDF212 /* route-with-straight-roundabout.json in Resources */,
+				E27A20D22654EBED00AA935F /* turn_left.data in Resources */,
```

Comment:
These test fixtures were added in #1813, but I don’t see how they were ever used in this project.

Reply:
Ok, removing.